### PR TITLE
Fix #3733 - stop spinner on json export

### LINKED ISSUE #3733: Export phenopacket JSON: spinner does not stop on download
Perhaps download to a _blank page or disable.. 
Screenshot 2022-12-20 at 15 15 43

### PR DESCRIPTION
**Pull request by dnil**
This PR adds fixes a bug.

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. add a phenotype to a case, and export phenopacket json
2. notice how before patch the spinner does not stop
3. after patch, the spinner stops

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by CR
- [x] tests executed by CR, DN

### CODE REVIEW

**Comment by northwestwitch** [comment]
I was trying to reproduce the error, but I don't see any spinner, it just downloads and that's it? Tested on both chrome and safari

**Comment by dnil** [comment]
Before patch? Nah, empty cache maybe? It is still the same on current main. See screenshot on the Issue, or a current one here:
Screenshot 2022-12-21 at 08 56 59

**Comment by northwestwitch** [comment]
> Before patch? Nah, empty cache maybe? It is still the same on current main. See screenshot on the Issue, or a current one here: Screenshot 2022-12-21 at 08 56 59

Perhaps when you have many phenotypes? I'll try to load some more!

**Comment by dnil** [comment]
Same in chrome and light mode. As you said, probably a dl json OS setting!

Screenshot 2022-12-21 at 09 08 00

File: scout/server/blueprints/cases/templates/cases/case.html [hunk 1]
```diff
@@ -596,6 +597,11 @@ <h5 class="modal-title" id="exampleModalLabel">Edit synopsis</h5>
 
   });
 
+function StopSpinner() {
```
Comment (northwestwitch) [inline scout/server/blueprints/cases/templates/cases/case.html:600]: 👍🏻

**Review by northwestwitch — APPROVED** [review]
It works perfectly! 💯

**Comment by dnil** [comment]
And for the record same in Edge win10 Enterprise.

**Pull request merged**